__tests__/app.test.js

```diff
@@ -88,6 +82,28 @@ describe("GET /api/articles/:article_id", () => {
         expect(response.body.status).toBe(400);
         expect(response.body.msg).toBe("bad request - invalid input type");
       });
+  });
+});
 
+describe("GET /api/users", () => {
+  it("should return an array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        expect(Array.isArray(response.body)).toBe(true);
+      });
+  });
+  it("should return an array of user objects with the correct keys", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const userArr = response.body;
```

Comment:
I'd recommend trying to destructure the value from the response.body eg `const { users } = response.body` - it gives your tests an added bit of security, and makes sure we're sending back the users on the correct key 😄